ci/k8s-upgrade/main.tf

```diff
@@ -1,10 +1,40 @@
-data "google_container_engine_versions" "central1b" {
-  provider       = google-beta
-  location       = "europe-west6"
+locals {
   version_prefix = "1.28."
   project        = "galoy-infra-testflight"
 }
 
+data "google_container_engine_versions" "euwest6" {
```

Comment:
Wouldn't a for-each work with a list of `locations`?

Reply:
yes - I didn't think about that
wanted to keep it simple